python/usb_console/run_simulation.py

```diff
@@ -118,7 +119,7 @@ def set_up_radios(self):
                 radio_session = RadioSession(radio_connected_device, radio_datastore, radio_logger)
 
                 if radio_session.connect(radio['imei']):
```

Comment:
It would also be good if the radio IMEI number came from your confidential.json file, rather than the config files that are currently publicly available. Can you modify the run_simulation logic for this?